lib/core/test/unit/Cardano/WalletSpec.hs

```diff
@@ -1330,9 +1331,19 @@ newtype DummyState
     = DummyState (Map Address (Index 'Soft 'AddressK))
     deriving (Generic, Show, Eq)
 
-instance Sqlite.PersistState DummyState where
-    insertState _ _ = error "DummyState.insertState: not implemented"
-    selectState _ = error "DummyState.selectState: not implemented"
+instance Sqlite.AddressBookIso DummyState where
```

Comment:
At some point, at other PR,  maybe we could have more useful DummyState instances here and get additional confidence by adding tests? Do you see something potentially useful here?

Reply:
As far as I understand, the purpose of the `WalletSpec` module is to test that the different layers work well together; however, the individual layers such as the DB layer are tested in other modules. So, I wouldn't make `DummyState` more fancy here, but rather ensure that we have sufficient testing of the individual layers.

In this case, I was wondering whether it was worth testing that the instances of `AddressBookIso` are indeed isomorphisms, but then decided against it, as the will-test-that code is too close to the to-be-tested code, and the current `SqliteSpec` state machine tests would highlight any issues here as well.

Reply:
ok, fair enough